settings.gradle

```diff
@@ -28,10 +28,12 @@ includeBuild("build-logic")
 
 // Libraries
 library("core")
+
+library("command")
```

Comment:
The rest of this list of libraries *seems* to be following alphabetical order. Not sure if it's coincidence but if it's not, this is out of order

Reply:
Damn it me.